### PR TITLE
Word twisted

### PR DESCRIPTION
**Pull request by jenshb**

### CODE REVIEW

**Comment by koevskinikola** [comment]
Hi @jenshb,

Can you provide some more information on how your change makes the information clearer?

Best,
Nikola

**Comment by jenshb** [comment]
I'm not an English expert, but in my mind the sentence in its current version is grammtically wrong. I had to read the sentence several times to try to understand its meaning.

Does it say: 
service calls (which might cause async) are made error
or       
service calls (which might cause async error) are made <missing>

I guess its meaning is:

service calls (which might cause an error) are made async

File: content/user-guide/process-engine/error-handling.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Let's show this in a concrete example: the user gets an error dialog on the fron
 
 ## Async and Failed Jobs
 
-If you don't want the exception being shown to the user, one option is to make service calls which might cause an async error as described in [Transactions in Processes]({{< ref "/user-guide/process-engine/transactions-in-processes.md" >}}). In that case the exception is stored in the process engine database and the [Job]({{< ref "/user-guide/process-engine/the-job-executor.md" >}}) in the background is marked as failed (to be more precise, the exception is stored and some retry counter is decremented).
+If you don't want the exception being shown to the user, one option is to make service calls which might cause an error async as described in [Transactions in Processes]({{< ref "/user-guide/process-engine/transactions-in-processes.md" >}}). In that case the exception is stored in the process engine database and the [Job]({{< ref "/user-guide/process-engine/the-job-executor.md" >}}) in the background is marked as failed (to be more precise, the exception is stored and some retry counter is decremented).
```
Comment (koevskinikola) [inline content/user-guide/process-engine/error-handling.md]:
```suggestion
If you don't want the exception being shown to the user, one option is to make service calls, which might cause an error, async (as described in [Transactions in Processes]({{< ref "/user-guide/process-engine/transactions-in-processes.md" >}})). In that case the exception is stored in the process engine database and the [Job]({{< ref "/user-guide/process-engine/the-job-executor.md" >}}) in the background is marked as failed (to be more precise, the exception is stored and some retry counter is decremented).
```

**Comment by koevskinikola** [comment]
@jenshb, thanks for the contribution! I just merged the changes with our codebase.

Best,
Nikola

**Pull request merged**